api/python/t4/api.py

```diff
@@ -449,3 +450,121 @@ def config(*autoconfig_url, **config_values):
         write_yaml(local_config, CONFIG_PATH, keep_backup=True)
 
     return HeliumConfig(CONFIG_PATH, local_config)
+
+def create_role(name, arn):
+    """
+    Create a new role in your registry. Admins only.
+
+    Required Parameters:
+        name(string): name of role to create
+        arn(string): ARN of IAM role to associate with the Quilt role you are creating
+    """
+    session = get_session()
+    response = session.post(
+        "{url}/api/roles".format(
+            url=get_registry_url()
+            ),
+            data=json.dumps({
+                'name': name,
+                'arn': arn
+            })
+        )
+    if not response.ok:
+        message = response.json()['message']
+        raise QuiltException("Error creating role: " + message)
+
+    return response.json()
+
+def edit_role(role_id, new_name=None, new_arn=None):
+    """
+    Edit an existing role in your registry. Admins only.
+
+    Required parameters:
+        role_id(string): ID of role you want to operate on.
+
+    Optional paramters:
+        new_name(string): new name for role
+        new_arn(string): new ARN for IAM role attached to Quilt role
+    """
+    session = get_session()
+    old_data = get_role(role_id)
+    data = {}
+    data['name'] = new_name or old_data['name']
+    data['arn'] = new_arn or old_data['arn']
+
+    response = session.put(
+        "{url}/api/roles/{role_id}".format(
+            url=get_registry_url(),
+            role_id=role_id
+            ),
+            data=data
```

Comment:
Hm... No `json.dumps`?

Though actually, you should be able to use `json=` instead of `data=`